### DISCUSSION TITLE
A WIP djot formater based on jotdown

### DISCUSSION THREAD

**Discussion by black-desk** [jgm/djot#328] [Show and tell]
https://github.com/black-desk/djotfmt

**Comment by jgm** [comment]
Does this behave differently from the djot formatter built into the javascript implementation?
i.e. `djot -f djot -t djot`?

**Reply by black-desk** [reply]
Sure, for now, it keep comments in formatted output.

https://github.com/black-desk/djotfmt/blob/master/tests/data/keep-comments/input.dj

https://github.com/black-desk/djotfmt/blob/master/tests/data/keep-comments/expected.dj
